packages/model/tests/workspace.test.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  Workspace,
+  WorkspaceOptions,
+  ProjectKind,
+  Service,
+  WebApp,
+  Package,
+} from '..';
+import {
+  createTestWebAppWorkspace,
+  createTestPackageWorkspace,
+  createTestServiceWorkspace,
+  createTestComboWorkspace,
+  createTestEmptyWorkspace,
+} from './utilities';
+
+// Test creation of different types of workspaces
+describe('Workspace', () => {
+  const name = 'test';
+  const root = '../../';
+
+  it('creates a workspace for a WebApp', () => {
+    const webAppWorkspace = createTestWebAppWorkspace(name, root);
+    expect(webAppWorkspace.name).toBe(name);
```

Comment:
I don't think there's much value in the `expect(thing.name).toBe(name);` in here or the other following tests. Lets remove it. Setting `name` is the responsibility of the `Base` class. That's not what we are testing here so it's just redundant assertions.